packages/zipkin-instrumentation-restify/src/restifyMiddleware.js

```diff
@@ -82,13 +82,19 @@ module.exports = function restifyMiddleware({tracer, serviceName = 'unknown', po
         tracer.recordBinary(Header.Flags, id.flags.toString());
       }
 
-      res.on('finish', () => {
+      const onCloseOrFinish = () => {
+        res.removeListener('close', onCloseOrFinish);
```

Comment:
Do we need to remove the listeners? My understanding is that `res.once(...)` will remove them for you, after invoking them?